docs/Design/Exceptions/Exception Handling.md

```diff
@@ -0,0 +1,286 @@
+# Exception Handling
+
+This doc will cover both try/catch/finally logic.
+Try blocks contain code that might throw exceptions.
```

Comment:
I still don't completely like this sentence. The following is totally valid:

```typescript
try { }
catch (error) { }
```

It's inaccurate to say that try blocks  contain code that might throw exceptions. They may also contain code that will never throw.